src/Frame.php

```diff
@@ -41,22 +51,24 @@ public function __construct(
         $this->class = $class;
 
         $this->applicationFrame = $isApplicationFrame;
+
+        $this->snippet = $snippet;
```

Comment:
Maybe rename this to `textSnippet`? To make it really clear that this isn't a `CodeSnippet` object.